Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -160,6 +160,7 @@ extension BrowserViewController: WKNavigationDelegate {
                 if let mainDocumentURL = navigationAction.request.mainDocumentURL, url.scheme == "http" {
                     let domainForShields = Domain.getOrCreate(forUrl: mainDocumentURL)
                     let isPrivateBrowsing = PrivateBrowsingManager.shared.isPrivateBrowsing
+                    // Loading bad data
```

Comment:
What does it mean?

Reply:
Removed, shouldn't have been committed. Thanks for catching.